.gitignore

```diff
@@ -129,6 +129,16 @@ ENV/
 # vim
 *.swp
 
+# Emacs
+*~
+\#*\#
+/.emacs.desktop
```

Comment:
You can (and should) configure your git with a global git exclude list so you don't need every project to support your choice of editor. https://git-scm.com/docs/gitignore

> 

Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

Reply:
OK, it's reasonable. So why `.swp` is included?

Reply:
Cos people don't know about this. I'm okay accepting this PR, just the more people I let know the easier their life is in the long run :)

Reply:
Yeah, I didn't know about that and got a better way. Thanks!

Reply:
And what does not help is most of the scaffolding where you generate .gitignore files already contains most of the "common" exclusions so adding them elsewhere seems inconsistent even if it avoids repetition.